packages/strapi-plugin-settings-manager/admin/src/translations/fr.json

```diff
@@ -65,14 +69,17 @@
   "request.error.environment.required": "L'environnement est obligatoire.",
   "request.error.environment.unknow": "L'environnement est inconnu.",
   "request.error.languages.exist": "Cette langue éxiste déjà.",
```

Comment:
`éxiste` should be replaced by `existe`.  This comment is also available for the `request.error.languages.exist` key.